docs/Using-Fleet/MDM-setup.md

```diff
@@ -53,53 +102,97 @@ Fleet UI:
 
 How to renew the certificate if it's expired or about to expire:
 
-1. Run the `fleetctl generate mdm-apple --email <email> --org <org>` command. Make sure you use the same Apple ID email address that you used when generating the original certificate.
+1. Run the `fleetctl generate mdm-apple --email <email> --org <org>` command. 
 
-2. Sign in to [Apple Push Certificates Portal](https://identity.apple.com) using the same Apple ID you used to get your original certificate. If you don't use the same Apple ID, you will have to turn MDM off and back on for all macOS hosts.
+2. Sign in to [Apple Push Certificates Portal](https://identity.apple.com) using the same Apple ID you used to get your original certificate. 
 
-3. In the **Settings > Integrations > Mobile device management (MDM)** page, under Apple Push Certificates portal, find the serial number of your current certificate. In Apple Push Certificates Portal, click  **Renew** next to the certificate that has the matching serial number. If you don't renew and get a new certificate, you will have to turn MDM off and back on for all macOS hosts.
+3. In the **Settings > Integrations > Mobile device management (MDM)** page, under Apple Push Certificates portal, find the serial number of your current certificate. In Apple Push Certificates Portal, click  **Renew** next to the certificate that has the matching serial number. 
```

Comment:
@noahtalerman There's more to this, isn't there? You need to go get a new CSR, then re-upload it and get a new cert then put the new certs into the Fleet server env vars? 

This is what I get when I tried it with a restaurant MDM: 
![image](https://user-images.githubusercontent.com/5359586/229250212-d6ba61d3-5177-471f-a496-c5780480d920.png)

Reply:
> You need to go get a new CSR, then re-upload it and get a new cert then put the new certs into the Fleet server env vars?

@zhumo that's right. I think this is partially covered by step (1):

> Run the `fleetctl generate mdm-apple --email <email> --org <org>` command. 

That said, I think we should be more explicit about the required files for APNs renewal. Something like 

```
### Step 1: generate required files
To renew APNs, we need to generate the two following files:
1. New APNs certificate 
2. New APNs private key 

<Fleet UI and fleetctl instructions for generating APNs CSR and key>

You can ignore the SCEP certificate and SCEP key. You don't need these to renew APNs.
```

And for step 2, renew certificate, we should be more explicit:

```
### Step 2: renew APNs certificate in Apple Push Certificates Portal

1. Log in to or enroll in [Apple Push Certificates Portal](https://identity.apple.com) using the same Apple ID you used to get your original APNs certificate.
2. In the Fleet UI, in the **Settings > Integrations > Mobile device management (MDM)** page, under Apple Push Certificates portal, find the serial number of your current certificate. 
3. In Apple Push Certificates Portal, click  **Renew** next to the certificate that has the matching serial number. 
4. Upload your CSR.
5. Download the new APNs certificate. 
```

And step 3, configure Fleet:

```
### Step 3: configure Fleet with the required files

With the two generated files, we now give them to the Fleet server. 

Restart the Fleet server with the contents of the APNs certificate and APNs private key in following environment variables:
* [FLEET_MDM_APPLE_APNS_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-cert-bytes)
* [FLEET_MDM_APPLE_APNS_KEY_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-key-bytes)

> You do not need to provide the APNs CSR which was emailed to you or the SCEP certificate and SCEP key which were generated in step 1.

Confirm that Fleet is set up by visiting the "Fleet UI" or using "fleetctl."

Fleet UI:

1. Head to the **Settings > Integrations > Mobile device management (MDM)** page.

2. Look at the **Apple Push Certificates Portal** section.

2. Follow the on-screen instructions.
`fleetctl` CLI:

1. Run `fleetctl get mdm-apple`.

You should see information about the new APNs certificate such as serial number and renewal date. 
```